connect_labs/mcp/tools/targeting.py

```
@@ -0,0 +1,383 @@
+        raise MCPToolError("BAD_REQUEST", f"Unknown basis {basis!r}") from None
+
+    cases_measure = interventions_mod.measure_for(unit_basis, indicator)
+    if cases_measure is None:
+        raise MCPToolError(
+            "BAD_REQUEST",
+            f"A {basis!r} basis has no count for {indicator!r} — that indicator implies no "
+            "case count, so pricing per case would be an approximation dressed as a figure. "
+            "Choose person, household, birth or under_5.",
+        )
+
+    selection, measure, chosen = _selection(
```

Ruff (0.16.2)

[warning] Unpacked variable `measure` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)